### PR TITLE
refresh the Dell iDRAC documentation

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

**Comment by YanMeddour** [comment]
the command output is empty, should we put an output such as: `OK: All hardware components are OK | temperature=25C;30;40 voltage=5V;;;; amperage=1A;;;;`

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md [hunk 1]
```diff
@@ -2,34 +2,363 @@
...
+
+<Tabs groupId="sync">
+<TabItem value="GlobalStatus" label="GlobalStatus">
+
+| Métrique       | Unité |
+|:---------------|:------|
+| status         | N/A   |
+| storage-status | N/A   |
+
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md:54]:
```suggestion
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md [hunk 2]
```diff
@@ -2,34 +2,363 @@
...
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--component='.*' \
+	--verbose
+```
+
+La commande devrait retourner un message de sortie similaire à :
+
+```bash
+OK: |
```
Comment (omercier) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md:231]: Not OK :-/

File: pp/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md [hunk 3]
```diff
@@ -2,34 +2,361 @@
...
+
+<Tabs groupId="sync">
+<TabItem value="GlobalStatus" label="GlobalStatus">
+
+| Metric name    | Unit  |
+|:---------------|:------|
+| status         | N/A   |
+| storage-status | N/A   |
+
+> To obtain this new metric format, include **--use-new-perfdata** in the **EXTRAOPTIONS** service macro.
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md:53]:
```suggestion
```

File: pp/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md [hunk 4]
```diff
@@ -2,34 +2,361 @@
...
+	--snmp-version='2c' \
+	--snmp-community='my-snmp-community'  \
+	--component='.*' \
+	--verbose
+```
+
+The expected command output is shown below:
+
+```bash
+OK: |
```
Comment (omercier) [inline pp/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md:229]: Same as in French version. The metrics names should be findable in the code.

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md [hunk 5]
```diff
@@ -2,34 +2,361 @@
...
+
+Le connecteur apporte les modèles de service suivants
+(classés selon le modèle d'hôte auquel ils sont rattachés) :
+
+<Tabs groupId="sync">
+<TabItem value="HW-Server-Dell-iDrac-SNMP-custom" label="HW-Server-Dell-iDrac-SNMP-custom">
+
+| Alias        | Modèle de service                      | Description                             |
+|:-------------|:---------------------------------------|:----------------------------------------|
+| GlobalStatus | HW-Dell-iDrac-GlobalStatus-SNMP-custom | Contrôle l'état globale de l'équipement |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md:24]:
```suggestion
| GlobalStatus | HW-Dell-iDrac-GlobalStatus-SNMP-custom | Contrôle l'état global de l'équipement |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md [hunk 6]
```diff
@@ -2,34 +2,361 @@
...
+| Hardware     | HW-Dell-iDrac-Hardware-SNMP-custom     | Contrôle l'état globale de l'équipement |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-servers-dell-idrac-snmp.md:25]:
```suggestion
| Hardware     | HW-Dell-iDrac-Hardware-SNMP-custom     | Contrôle l'état global de l'équipement |
```

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**